index.html

```diff
@@ -228,36 +257,36 @@
         example including scripting, are presented.
       </p>
       <ul>
-        <li id="example-1">To take a picture using the device's local still
-        image capture device, such as a camera, and upload the picture taken
-        using an HTML form:
+        <li id="example-1">To take a picture using the device's user-facing
+        camera, and upload the picture taken using an HTML form:
           <pre class="example">
           &lt;form action="server.cgi" method="post" enctype="multipart/form-data"&gt;
-            &lt;input type="file" name="image" accept="image/*" capture&gt;
+            &lt;input type="file" name="image" accept="image/*" capture="user"&gt;
             &lt;input type="submit" value="Upload"&gt;
           &lt;/form&gt;
         </pre>
         </li>
         <li>Or alternatively, to capture video using the device's local video
-        camera:
+        camera facing the environment:
```

Comment:
Also it's in the queue of caniuse (https://github.com/Fyrd/caniuse/issues/1115) waiting for some candid soul to add the entry (as a reference, this is the [`mediacapture-image`](https://github.com/Fyrd/caniuse/pull/2738)).